posts/self-hosted-homelab/index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Run Coder in a self-hosted homelab
+date: April 7, 2023
+author: Marc Paquette
+---
+
+# Run Coder in a self-hosted homelab
+
+I outgrew my little homelab. I have a couple of [ODROID-HC2](https://ameridroid.com/products/odroid-hc2) devices in a closet. 
+
+I use them to serve files and experiment with ARM assembly. They work great but they’re small, single board computers with equally small capacity.
+
+It was time to expand my homelab to handle my dev projects too.
+
+## My problem: I wasn't getting anywhere
+
+I was spending too much time managing my personal and work projects instead of working on them. They're spread out over several environments and places. They run on OpenBSD, Windows, Linux, ARM, python3, browsers, and old-school command-line tools. The ODROIDs are almost literally chained to my homelab closet. I work from home, a co-work space, and sometimes other time zones.
+
+The ideal solution would let me self-host my projects and work on them wherever and however I want. The machine I’m typing on right now is an OpenBSD laptop. I sometimes use a Windows laptop. I run Linux on the ODROIDs. And I also wanted to use an iPad for the [ultimate sofa software development rig](https://coder.com/blog/a-guide-to-writing-code-on-an-ipad).
+
+I put off fixing this because I expected that I'd have to hack and contort different tools into something that might eventually approach what I wanted. The cure seemed worse than the disease.
+
+## I discovered Coder
+
+It turns out that [Coder](https://coder.com) is a much easier solution. Coder solves big problems for big enterprise dev teams. And I discovered that it can solve my problem too:
```

Comment:
I'd suggest linking to our GitHub or documentation versus our website. It's much less enterprise-y. Your call

- https://coder.com/docs/v2
- https://github.com/coder/coder

Reply:
Good idea. Thanks.